rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -154,4 +155,23 @@ private static void signLegacyTransactionInputFromP2shMultiSig(BtcTransaction tr
             input.setScriptSig(inputScriptSig);
         }
     }
+
+    public static List<Sha256Hash> generateTransactionInputsSigHashes(BtcTransaction btcTx) {
```

Comment:
```suggestion
    public static List<Sha256Hash> getInputSigHashes(BtcTransaction btcTx) {
```

Wdyt?

Reply:
i like better the `generate` since is what its doing